### ISSUE TITLE
Update references of the `monitoring` namespace to `knative-monitoring`

### ISSUE THREAD

**Issue by tanzeeb** [knative/docs#407]
Once https://github.com/knative/serving/pull/1851 is merged, we will need to update the documentation to reflect the new name change. 

All references to the `monitoring` namespace should be changed to `knative-monitoring`.

### What needs to be changed (not exhaustive):
* URLs (eg.  `/api/v1/namespaces/monitoring/` -> `/api/v1/namespaces/knative-monitoring/`)
* Command line flags (eg. `kubectl get pods --namespace monitoring --watch` -> `kubectl get pods --namespace knative-monitoring --watch`)

### What does *not* need to be changed:
* File paths (eg. `kubectl apply -Rf config/monitoring/` does not need to change)

**Comment by tanzeeb** [comment]
Making a list:
- [ ] serving/accessing-logs.md
- [ ] serving/debugging-application-issues.md
- [ ] install/Knative-with-OpenShift.md
- [ ] serving/installing-logging-metrics-traces.md
- [ ] serving/accessing-metrics.md
- [ ] serving/setting-up-a-logging-plugin.md
- [ ] install/scripts/knative-with-openshift.sh
- [ ] serving/samples/telemetry-go/README.md
- [ ] serving/samples/telemetry-go/telemetrysample.go
- [ ] serving/samples/autoscale-go/README.md
- [ ] eventing/samples/k8s-events/README.md

**Comment by RichieEscarez** [comment]
FYI: I am currently editing a few the logging related files, including:

serving/installing-logging-metrics-traces.md 
serving/accessing-metrics.md
serving/setting-up-a-logging-plugin.md

You can consider these done/in-progress. Please let me handle these changes since the files have changed significantly and a separate PR will cause merge conflicts.

**Comment by tanzeeb** [comment]
Hey @RichieEscarez, no problem! Should I submit a PR for the other changes, or leave them?

**Comment by RichieEscarez** [comment]
I was actually wonder when the code related changes will actually work in someones install?
For example, when will the commands/URLs in the docs actually work (I'd like to make sure the docs include "working commands" until the changes actual appear in the product)?

That said, im happy to take this on and handle the other files in a separate PR.

**Comment by yanweiguo** [comment]
If someone installs knative from the release version, which currently is https://github.com/knative/serving/releases/download/v0.1.1/release.yaml, then the code relative changes will actually work after next release which includes the changes.

If someone installs knative serving from master branc(as a developer), then the code relative changes should actually work.
